api/app/views/mno_enterprise/jpi/v1/teams/_team.json.jbuilder

```diff
@@ -10,6 +10,7 @@ end
 json.app_instances do
   json.array! team.app_instances do |instance|
     json.extract! instance, :id, :name
-    json.logo instance.app.logo
+    # Check if app is present as it is not needed for create/update actions
+    json.logo instance.app.logo if instance.app
```

Comment:
why are we calling #to_s in the admin jbuilder and not here?

Reply:
Good question, going to remove the `to_s` in the admin view, I don't see why it would be there honestly